packages/valory/skills/registration_abci/fsm_specification.yaml

```diff
@@ -6,7 +6,7 @@ default_start_state: RegistrationStartupRound
 final_states:
 - FinishedRegistrationFFWRound
 - FinishedRegistrationRound
-label: packages.valory.skills.registration_abci.rounds.AgentRegistrationAbciApp
+label: AgentRegistrationAbciApp
```

Comment:
We don't need entire path to the class definition, it makes working with the app class definition in the command like harder

Reply:
This label is mostly ignored by the FSM speck checker. It can be anything. I don't see any harm in keeping the whole classpath.

Reply:
It maybe ignored by the spec checker but it is particularly helpful when we want to run a check. When running we can just pick the class name from here and use it to import the module, the user can choose to provide the class name but if they do not we can just use this